build/appveyor.yml

```diff
@@ -70,19 +72,19 @@ test_script:
 after_test:
 - ps: ./build/prepare-deploy.ps1
 
-deploy:
-  # Deploy to GitHub Releases
-  - provider: GitHub
-    auth_token:
-      secure: IGs25EMrhe0au+uVDbOck6akDzC9ykvJeR0zJjhrT6idNuYOr/JpyGdefQxSlWXk
-    release: DesignGuide v$(GitVersion_SemVer)
-    tag: $(APPVEYOR_REPO_TAG_NAME)
-    description: $(changelog)
-    artifact: /PayEx.DesignGuide.*\.zip/
-    draft: false
-    prerelease: false
-    on:
-      appveyor_repo_tag: true
+# deploy:
+# Deploy to GitHub Releases
+- provider: GitHub
+  auth_token:
+    secure: IGs25EMrhe0au+uVDbOck6akDzC9ykvJeR0zJjhrT6idNuYOr/JpyGdefQxSlWXk
+  release: DesignGuide v$(GitVersion_SemVer)
+  tag: $(APPVEYOR_REPO_TAG_NAME)
+  description: $(changelog)
+  artifact: /SwedbankPay.DesignGuide.*\.zip/
+  draft: false
+  prerelease: false
+  on:
+    appveyor_repo_tag: true
```

Comment:
What is going on here? Did you mean to comment out the entire deploy-step?

Reply:
Comments left from previous tests.